Source/WebDriver/WebDriverService.cpp

```diff
@@ -124,13 +155,30 @@ int WebDriverService::run(int argc, char** argv)
         return EXIT_FAILURE;
     }
 
+#if ENABLE(WEBDRIVER_BIDI)
+    auto bidiPort = parseInteger<uint16_t>(bidiPortString);
+    if (!bidiPort) {
+        fprintf(stderr, "Invalid BiDi port %s provided. Defaulting to 4445.\n", bidiPortString.utf8().data());
```

Comment:
I would call it a WebSocket port in the logging, since BiDi could mean multiple things.

Reply:
Renamed.